### ISSUE TITLE
Common wirings

### ISSUE THREAD

**Issue by martincollado** [NHBSystems/NHB_AD7794#2]
Would be nice to have on the documentation some wiring diagrams for the connections of common sensors and some boards if not using the feather layout.

Sensors:
- Load cells
- Pressure Sensors
- Thermocouples

Boards:
- ESP32
- Arduino Nano
- Arduino Uno

**Comment by jjuliano77** [comment]
Good point. I will start putting something together this evening.

**Comment by SkaveRat** [comment]
any update on this?

I'm in the process on setting this up with a load cell and I'm not entirely sure how to connect it

**Comment by jjuliano77** [comment]
Yes, example sensor connections can be found here. [https://nhbsystems.com/reading-analog-sensors/](https://nhbsystems.com/reading-analog-sensors/). I have not done hookups for non-feather boards yet, however you really just need to connect power and the SPI bus (MISO, MOSI, SCK, and a CS line). If you want to control the excitation voltage from the microcontroller you will also need to connect the EX Enable line to an IO pin and cut the trace jumper.  

Also, the AD7794 technically _should_ be able to operate at 5v if using something like an UNO, but I have not tested it because the Feather format specifies a 3.3v interface.
